composer.json

```diff
@@ -19,7 +19,7 @@
 	},
 	"require": {
 		"luyadev/luya-core" : "^1.0",
-		"luyadev/luya-module-cms": "^1.0 | ^2.0",
+		"luyadev/luya-module-cms": "*",
```

Comment:
```suggestion
		"luyadev/luya-module-cms": "^1.0 | ^2.0 | ^3.0",
```

Reply:
sure, this works as well. We just have to send a PR on next release. If there is a BC break i would have to send a PR with adjustments anyhow, so the `*` would be fine i thought.